content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,60 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myconfigmap
+data:
+  username: "k8s-admin"
+  access_level: "1"
+```
+The following command will create the ConfigMap object:
```

Comment:
It would be great to delimit code-blocks from the text by new line ;-)

```suggestion

The following command will create the ConfigMap object:

```